liblangutil/ErrorReporter.h

```diff
@@ -183,9 +185,11 @@ class ErrorReporter
 
 	unsigned m_errorCount = 0;
 	unsigned m_warningCount = 0;
+	unsigned m_noticeCount = 0;
 
 	unsigned const c_maxWarningsAllowed = 256;
 	unsigned const c_maxErrorsAllowed = 256;
+	unsigned const c_maxNoticesAllowed = 256;
```

Comment:
If notices are non-harmful, why is there a limit?

Reply:
Just copied the others, I'm fine with either.

Reply:
It's not about being harmful

Reply:
> If notices are non-harmful, why is there a limit?

It is there to be deterministic, otherwise certain inputs could trigger large/infinite messages and the output depends on the platform/machine.

There is even a test case for this, which was found by some fuzzer or audit.

Reply:
I think that this should be explained in a comment above these. It is not obvious why we have these limits.